elasticai/creator/nn/quantized_grads/base_modules/linear.py

```diff
@@ -0,0 +1,40 @@
+from typing import Any, Callable, Protocol
+
+import torch
+
+from elasticai.creator.base_modules.math_operations import Add, MatMul, Quantize
+
+
+class MathOperations(Quantize, Add, MatMul, Protocol): ...
+
+
+class Linear(torch.nn.Linear):
+    """This module implements a linear layer.
```

Comment:
Imho you could remove the comment that states that this is a linear layer, cause that's clear from context and name. But maybe that's just me, so do as you like it.